test/app/android/app/src/main/java/com/example/twiliovoicereactnative/MainReactNativeHost.java

```diff
@@ -1,30 +0,0 @@
-package com.example.twiliovoicereactnative;
```

Comment:
Why did you pull this out. The design now requires the customer to implement this every time...

Reply:
Discussed over a call, this is autogenerated by the RN CLI at bootstrap when the app is made.